src/musig.rs

```diff
@@ -129,32 +140,28 @@ impl fmt::Display for InvalidTweakErr {
 ///
 /// Remember that nonce reuse will immediately leak the secret key!
 ///
-/// # Errors:
-///
-/// * `ZeroSession`: if the `session_secrand` is supplied is all zeros.
-///
 /// Example:
 ///
 /// ```rust
-/// # # [cfg(any(test, feature = "rand-std"))] {
-/// # use secp256k1::rand::{thread_rng, RngCore};
-/// # use secp256k1::{PublicKey, Secp256k1, SecretKey, new_nonce_pair, SessionSecretRand};
+/// # # [cfg(feature = "std")]
+/// # # [cfg(feature = "rand")] {
```

Comment:
In 48d5c8f69833ec2c249e9172117084d94e7b63c7 this syntax is odd, is it meant to eb `#![cfg(...)]`?

Reply:
Yeah, something is messed up with my rustfmt setup..

Reply:
Oh I bet that is why CI is failing, I didn't look though.

Reply:
No, there shouldn't be a `!`. But there shouldn't be a space eithr.